cmd/nullbootctl/main.go

```diff
@@ -116,4 +123,25 @@ func main() {
 			os.Exit(1)
 		}
 	}
+
+	if jsonEfivars := efivars.(*efibootmgr.MockEFIVariables); jsonEfivars != nil {
```

Comment:
```suggestion
		if jsonEfivars, ok := efivars.(*efibootmgr.MockEFIVariables); ok {
```
Otherwise, it panics:
```
panic: interface conversion: efibootmgr.EFIVariables is efibootmgr.RealEFIVariables, not *efibootmgr.MockEFIVariables

goroutine 1 [running]:
main.main()
        /home/ubuntu/nullboot/cmd/nullbootctl/main.go:127 +0x10ed
```